src/OrchardCore.Modules/OrchardCore.AdminTrees/Views/Items/LinkAdminNode.Fields.TreeSummary.cshtml

```diff
@@ -0,0 +1,12 @@
+@model ShapeViewModel<LinkAdminNode>
+
+@{
+    string linkText = Model.Value.LinkText;
+    string linkUrl = Model.Value.LinkUrl;
+    bool enabled = Model.Value.Enabled;
+}
+
+<div class="@(enabled ? "" : "disabled") concrete-link">
+    <h5>@T["Link"]</h5>
+    <span class="text-muted">@T["Link Text:"] @linkText | @T["Link Url:"]: @linkUrl</span>
```

Comment:
`@T["Link Text: {0}", linkText] | @T["Link Url: {0}", linkUrl]`

Because in a different language the order might matter.